deploy/common/k8sbased.sh

```diff
@@ -946,16 +946,12 @@ function launch_sensor {
         if [[ "${ROX_HOTRELOAD}" == "true" ]]; then
             hotload_binary bin/kubernetes-sensor kubernetes sensor "${sensor_namespace}"
         fi
-        if [[ -z "${IS_RACE_BUILD}" ]]; then
-           kubectl -n "${sensor_namespace}" patch deploy/sensor --patch '{"spec":{"template":{"spec":{"containers":[{"name":"sensor","resources":{"limits":{"cpu":"500m","memory":"500Mi"},"requests":{"cpu":"500m","memory":"500Mi"}}}]}}}}'
-        fi
+        kubectl -n "${sensor_namespace}" patch deploy/sensor --patch '{"spec":{"template":{"spec":{"containers":[{"name":"sensor","resources":{"limits":{"cpu":null,"memory":"500Mi"},"requests":{"cpu":"500m","memory":"500Mi"}}}]}}}}'
```

Comment:
```suggestion
        kubectl -n "${sensor_namespace}" patch deploy/sensor --patch '{"spec":{"template":{"spec":{"containers":[{"name":"sensor","resources":{"limits":{"cpu":null,"memory":"500Mi"},"requests":{"cpu":"500m","memory":null}}}]}}}}'
```

Reply:
I'll leave it as it is. I would prefer to have request and limit explicitly defined.